src/main/scala/org/apache/spark/sql/hive/llap/LlapMetastoreCatalog.scala

```diff
@@ -49,6 +53,7 @@ class LlapMetastoreCatalog(sparkSession: SparkSession)
         className = "org.apache.spark.sql.hive.llap",
         options = Map(
           "table" -> (qualifiedTableName.database + "." + qualifiedTableName.name),
+          "sizeinbytes" -> size.toString(),
```

Comment:
Does this need to be checked to be a valid value like >= 0 or does the receiving code handle that?